data-loader/cli/src/main/java/com/scalar/db/dataloader/cli/util/CommandLineInputUtils.java

```diff
@@ -0,0 +1,48 @@
+package com.scalar.db.dataloader.cli.util;
+
+import com.scalar.db.common.error.CoreError;
+import com.scalar.db.dataloader.core.ColumnKeyValue;
+import java.util.Objects;
+
+public class CommandLineInputUtils {
+
+  /**
+   * Parses a single key-value pair from a string in the format "key=value".
+   *
+   * @param keyValue the key-value string to parse
+   * @return a {@link ColumnKeyValue} object representing the parsed key-value pair
+   * @throws IllegalArgumentException if the input is null, empty, or not in the expected format
+   */
+  public static ColumnKeyValue parseKeyValue(String keyValue) {
```

Comment:
This class is `CommandLineInputUtils` and the method name is `parseKeyValue()`. So, it doesn't seem to return the specific type `ColumnKeyValue`. How about returning more generic type like `Pair<K, V>` or `Tuple<K, V>` (or Map.Entry<K, V>?) ?

Reply:
@komamitsu san,
I have updated the util method implementation and also made changes to use the function to parse the scan order.
Thank you.